### PR TITLE
add contributing.md and changed readme.md

### PR DESCRIPTION
**Pull request by n3k00n3**
I added a contributing file to help new contributors.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+1. [Fork](https://help.github.com/articles/fork-a-repo) the main repository.
+   This is your own copy of the `virgulino-ruby` project to work in.
+2. Clone your repository to your local machine.
+3. `git checkout -b newbranch`
+This creates a new branch, called `newbranch` in our example, in your local
+repository.
+4. Make your changes.
+5. `git commit`
+6. `git push origin newdesign:newdesign`
```
Comment (UserXGnu) [inline CONTRIBUTING.md]: if the branch created is `newbranch' the push must be on `newbranch`  branch, so the rightway is:
git add --all
git commit -am "A comment based on the featured added or fixing matters" 
git push -u origin newbranch

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+This pushes your new branch called `newdesign` to your GitHub repository.
```
Comment (UserXGnu) [inline CONTRIBUTING.md]: doing that way, would be fine to remove that line of explanation.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+## Integrating your working code to master
+
+When you have made your changes and tested them, please send us a pull request.
```
Comment (UserXGnu) [inline CONTRIBUTING.md]: s/when you have/once you/g

Comment (n3k00n3) [inline CONTRIBUTING.md]: Done!

**Pull request merged**